presto-hive/pom.xml

```diff
@@ -204,7 +204,7 @@
         <dependency>
             <groupId>com.facebook.presto</groupId>
             <artifactId>presto-main</artifactId>
-            <scope>test</scope>
+            <scope>provided</scope>
```

Comment:
get it. Really need RowType and RowField, which are in presto-main, not in spi. Is there any special reason RowType (MapType, etc.) is not in spi? Could we move RowType in spi? Or, maybe  duplicate the code in presto-hive? @martint @cberner @dain